webapp/components/Comment.vue

```diff
@@ -39,19 +39,28 @@
         />
       </div>
       <div v-show="!openEditCommentMenu">
-        <div v-if="isCollapsed" v-html="comment.contentExcerpt" style="padding-left: 40px;" />
+        <content-viewer
+          v-if="comment.content.length < 180"
+          :content="comment.content"
+          class="padding-left"
+        />
         <div
           v-show="comment.content !== comment.contentExcerpt && comment.content.length > 180"
-          style="text-align: right;  margin-right: 20px; margin-top: -12px;"
+          class="show-more-or-less-div"
         >
+          <content-viewer
+            v-if="isCollapsed"
+            :content="comment.contentExcerpt"
+            class="padding-left text-align-left"
+          />
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-09-17T16:43:41Z" title="Tuesday, September 17th 2019, 6:43:41 pm +02:00">Sep 17, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -39,18 +39,27 @@
         />
       </div>
       <div v-show="!openEditCommentMenu">
-        <div v-if="isCollapsed" v-html="comment.contentExcerpt" style="padding-left: 40px;" />
+        <content-viewer
+          v-if="comment.content.length < 180"
+          :content="comment.content"
+          class="padding-left"
+        />
         <div
           v-show="comment.content !== comment.contentExcerpt && comment.content.length > 180"
           style="text-align: right;  margin-right: 20px; margin-top: -12px;"
         >
+          <content-viewer
+            v-if="isCollapsed"
+            :content="comment.contentExcerpt"
+            class="padding-left"
+          />
```


@mattwr18 I don't understand why we have `content` twice now, and the `contentExcerpt` here..